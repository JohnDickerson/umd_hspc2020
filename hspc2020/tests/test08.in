11
6 4 .
6 4 V 2 . .
6 4 V 2 . H 3 . .
6 4 V 2 H 1 . . H 3 V 2 . . . 
10 10 V 5 V 2 V 1 . . V 2 V 1 . . . H 9 H 8 H 7 H 6 H 5 H 4 H 3 H 2 H 1 . . . . . . . . . .
20 10 H 5 H 1 . H 1 . H 1 . H 1 . . V 8 . H 1 . H 1 . H 1 . H 1 . . 
10 10 H 1 . V 9 H 8 V 1 . H 1 . V 7 H 6 V 1 . H 1 . V 5 H 4 V 1 . H 1 . V 3 H 2 V 1 . H 1 . V 1 . . . . . . . . . .
20 1 V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . V 1 . .
1 20 H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . H 1 . .
20 1 V 18 V 16 V 14 V 12 V 10 V 8 V 6 V 4 V 2 . . . . . . . . . .
16 16 V 8 H 8 . V 4 H 4 . V 2 H 2 . V 1 H 1 . . H 1 . . H 2 V 1 H 1 . . H 1 . . . H 4 V 2 H 2 . V 1 H 1 . . H 1 . . H 2 V 1 H 1 . . H 1 . . . . H 8 V 4 H 4 . V 2 H 2 . V 1 H 1 . . H 1 . . H 2 V 1 H 1 . . H 1 . . . H 4 V 2 H 2 . V 1 H 1 . . H 1 . . H 2 V 1 H 1 . . H 1 . . . . .
