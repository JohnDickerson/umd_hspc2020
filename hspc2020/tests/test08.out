Drawing 1
+--+--+--+--+--+--+
|                 |
+  +  +  +  +  +  +
|                 |
+  +  +  +  +  +  +
|                 |
+  +  +  +  +  +  +
|                 |
+--+--+--+--+--+--+
Drawing 2
+--+--+--+--+--+--+
|     |           |
+  +  +  +  +  +  +
|     |           |
+  +  +  +  +  +  +
|     |           |
+  +  +  +  +  +  +
|     |           |
+--+--+--+--+--+--+
Drawing 3
+--+--+--+--+--+--+
|     |           |
+  +  +--+--+--+--+
|     |           |
+  +  +  +  +  +  +
|     |           |
+  +  +  +  +  +  +
|     |           |
+--+--+--+--+--+--+
Drawing 4
+--+--+--+--+--+--+
|     |           |
+  +  +--+--+--+--+
|     |     |     |
+  +  +  +  +  +  +
|     |     |     |
+--+--+  +  +  +  +
|     |     |     |
+--+--+--+--+--+--+
Drawing 5
+--+--+--+--+--+--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+  +  +  +  +  +--+--+--+--+--+
|  |  |  |  |  |              |
+--+--+--+--+--+--+--+--+--+--+
Drawing 6
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                       |                                   |
+  +  +  +  +  +  +  +  +--+--+--+--+--+--+--+--+--+--+--+--+
|                       |                                   |
+  +  +  +  +  +  +  +  +--+--+--+--+--+--+--+--+--+--+--+--+
|                       |                                   |
+  +  +  +  +  +  +  +  +--+--+--+--+--+--+--+--+--+--+--+--+
|                       |                                   |
+  +  +  +  +  +  +  +  +--+--+--+--+--+--+--+--+--+--+--+--+
|                       |                                   |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                                                           |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                                                           |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                                                           |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                                                           |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                                                           |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
Drawing 7
+--+--+--+--+--+--+--+--+--+--+
|                          |  |
+--+--+--+--+--+--+--+--+--+  +
|  |                    |  |  |
+  +--+--+--+--+--+--+--+  +  +
|  |  |              |  |  |  |
+  +  +--+--+--+--+--+  +  +  +
|  |  |  |        |  |  |  |  |
+  +  +  +--+--+--+  +  +  +  +
|  |  |  |  |  |  |  |  |  |  |
+  +  +  +  +--+--+  +  +  +  +
|  |  |  |  |     |  |  |  |  |
+  +  +  +--+--+--+--+  +  +  +
|  |  |  |           |  |  |  |
+  +  +--+--+--+--+--+--+  +  +
|  |  |                 |  |  |
+  +--+--+--+--+--+--+--+--+  +
|  |                       |  |
+--+--+--+--+--+--+--+--+--+--+
|                             |
+--+--+--+--+--+--+--+--+--+--+
Drawing 8
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
Drawing 9
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
|  |
+--+
Drawing 10
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|     |     |     |     |     |     |     |     |     |     |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
Drawing 11
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|  |  |     |           |                       |
+--+--+  +  +  +  +  +  +  +  +  +  +  +  +  +  +
|  |  |     |           |                       |
+--+--+--+--+  +  +  +  +  +  +  +  +  +  +  +  +
|     |  |  |           |                       |
+  +  +--+--+  +  +  +  +  +  +  +  +  +  +  +  +
|     |  |  |           |                       |
+--+--+--+--+--+--+--+--+  +  +  +  +  +  +  +  +
|           |  |  |     |                       |
+  +  +  +  +--+--+  +  +  +  +  +  +  +  +  +  +
|           |  |  |     |                       |
+  +  +  +  +--+--+--+--+  +  +  +  +  +  +  +  +
|           |     |  |  |                       |
+  +  +  +  +  +  +--+--+  +  +  +  +  +  +  +  +
|           |     |  |  |                       |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
|                       |  |  |     |           |
+  +  +  +  +  +  +  +  +--+--+  +  +  +  +  +  +
|                       |  |  |     |           |
+  +  +  +  +  +  +  +  +--+--+--+--+  +  +  +  +
|                       |     |  |  |           |
+  +  +  +  +  +  +  +  +  +  +--+--+  +  +  +  +
|                       |     |  |  |           |
+  +  +  +  +  +  +  +  +--+--+--+--+--+--+--+--+
|                       |           |  |  |     |
+  +  +  +  +  +  +  +  +  +  +  +  +--+--+  +  +
|                       |           |  |  |     |
+  +  +  +  +  +  +  +  +  +  +  +  +--+--+--+--+
|                       |           |     |  |  |
+  +  +  +  +  +  +  +  +  +  +  +  +  +  +--+--+
|                       |           |     |  |  |
+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
